### PR TITLE
Add jupyter notebook integration.

### LINKED ISSUE #2000: document jupyter notebook as an available integration
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

Similar to Neon's doc on jupyter notebook integration https://neon.tech/docs/ai/ai-google-colab, we only need to tell users that they can use postgresql python driver, such as psycopg2, to connect to risingwave. there is no difference from writing a normal python app. They can see https://docs.risingwave.com/docs/current/python-client-libraries/ for details. 

### Reference

_No response_

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **Description**

  - [ What's changed? Which parts of the docs are affected? ]

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/2000#event-12336961190

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Review by neverchanje — APPROVED** [review]

File: docs/rw-integration-summary.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ RisingWave can sink data to Kafka. This allows indirectly sinking data to any sy
 |Cube.js| | Available. See the [documentation of Cube](https://cube.dev/docs/product/configuration/data-sources/postgres) for details.|
 |DBeaver| | Available. See [Connect DBeaver to RisingWave](/guides/dbeaver-integration.md) for details. |
 |Grafana| |Available. See [Visualize RisingWave data in Grafana](/guides/grafana-integration.md) for details.|
-|Jupyter Notebook||Researching <voteNotify note="jupyter" />|
+|Jupyter Notebook||Available. For this integration, you can use a PostgreSQL python driver, such as Psycopg 2, to connect to risingwave. There is no difference from writing a normal python app. See [Use RisingWave in your Python application](/dev/go-client-libraries.md) for details.|
```
Comment (WanYixian) [inline docs/rw-integration-summary.md]:
```suggestion
|Jupyter Notebook||Available. For this integration, you can use a PostgreSQL Python driver, such as `psycopg2`, to connect to RisingWave. There is no difference from writing a normal python app. See [Use RisingWave in your Python application](/dev/python-client-libraries.md) for details.|
```

File: versioned_docs/version-1.8/rw-integration-summary.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ RisingWave can sink data to Kafka. This allows indirectly sinking data to any sy
 |Cube.js| | Available. See the [documentation of Cube](https://cube.dev/docs/product/configuration/data-sources/postgres) for details.|
 |DBeaver| | Available. See [Connect DBeaver to RisingWave](/guides/dbeaver-integration.md) for details. |
 |Grafana| |Available. See [Visualize RisingWave data in Grafana](/guides/grafana-integration.md) for details.|
-|Jupyter Notebook||Researching <voteNotify note="jupyter" />|
+|Jupyter Notebook||Available. For this integration, you can use a PostgreSQL python driver, such as Psycopg 2, to connect to risingwave. There is no difference from writing a normal python app. See [Use RisingWave in your Python application](/dev/go-client-libraries.md) for details.|
```
Comment (WanYixian) [inline versioned_docs/version-1.8/rw-integration-summary.md]:
```suggestion
|Jupyter Notebook||Available. For this integration, you can use a PostgreSQL Python driver, such as `psycopg2`, to connect to RisingWave. There is no difference from writing a normal python app. See [Use RisingWave in your Python application](/dev/python-client-libraries.md) for details.|
```

**Review by WanYixian — APPROVED** [review]
The link should direct to Python application, not Go, right? Others LGTM

File: docs/rw-integration-summary.md (hunk 1 above)
Comment (ShanlanLi) [reply docs/rw-integration-summary.md]: Yes. Good catch. Thanks!

**Pull request merged**